model/llm/prompt/parse.go

```diff
@@ -13,16 +14,27 @@ var (
 )
 
 // ParseResponse parses code from a model's response.
-func ParseResponse(response string) (code string) {
+func ParseResponse(response string) (assessment metrics.Assessments, code string) {
+	assessment = metrics.Assessments{}
+
 	// Some models produce duplicated code tags, so unify them if needed.
 	response = codeTagDuplicatedRe.ReplaceAllString(response, "```")
 
 	blocks := bytesutil.GuardedBlocks(response, codeTagRe, codeTagRe)
 	if len(blocks) == 0 { // When no code blocks are found, assume that just the code is returned.
```

Comment:
This and the next comment... sound like block comments. can you make blocks out of the code please.

```
blocks := bytesutil.GuardedBlocks(response, codeTagRe, codeTagRe)

// When no code blocks are found, assume that just the code is returned.
if len(blocks) == 0 { 
...

// Assume the first code block contains the response code fragment.
	block := blocks[0]

```